### ISSUE TITLE
Change Palettes' examples' title colors to be always readable

### ISSUE THREAD

**Issue by ajsecord** [material-components/material-components-ios#728]
The Catalog examples for Palettes puts black text on a variety of background colors, but for darker colors it's unreadable. 

Use MDFTextAccessibility to change the title colors appropriately to be readable.

**Comment by romoore** [comment]
Closed by #3590
